feature-libs/cart/import-export/components/import-export/import-export.component.html

```diff
@@ -0,0 +1,12 @@
+<ng-container *ngIf="cmsComponentData$ | async as data">
+  <ng-container *ngIf="route$ | async as route">
+    <cx-import-entries
+      *ngIf="shouldDisplayButton(data.importButtonDisplayPages, route)"
+      [cartType]="getCartType(data.routesCartMapping, route)"
+    ></cx-import-entries>
+    <cx-export-entries
+      *ngIf="shouldDisplayButton(data.exportButtonDisplayPages, route)"
+      [cartType]="getCartType(data.routesCartMapping, route)"
+    ></cx-export-entries>
+  </ng-container>
+</ng-container>
```

Comment:
Nitpick: A lot of knowledge about routes, cms data and configs internals is brought to the template. IMHO template doesn't need to know about them.

Alternatively you could expose to the template just 3 observables: `shouldDisplayImport$`, `shouldDisplayExport$`, `cartType$`.

I don't insist on refactoring. IMHO it would be just clearer division of responsibilities.

```typescript
shouldDisplayImport$: Observable<boolean> = combineLatest([ route$, this.cmsComponent.data$]).pipe(
  map(([route, data]) => data.importButtonDisplayRoutes.includes(route))
);

shouldDisplayExport$: Observable<boolean> = combineLatest([ route$, this.cmsComponent.data$]).pipe(
  map(([route, data]) => data.exportButtonDisplayRoutes.includes(route))
);

cartType$: Observable<CartType | undefined> = combineLatest([route$, this.cmsComponent.data$]).pipe(
   map(([route, data]) => data.routesCartMapping[route])
);
```


```html
<cx-import-entries
  *ngIf="shouldDisplayImport$ | async"
   [cartType]="cartType$"
></cx-import-entries>
<cx-export-entries
  *ngIf="shouldDisplayExport$ | async"
  [cartType]="cartType$ | async"
></cx-export-entries>
```